### PR TITLE
First attempt at updating config.json to support nextercism

### PR DESCRIPTION
**Pull request by ricemery**
First attempt at updating config.json to support nextercism.

seems like too much hooked up to `hamming` and `accumulate`

refs #398

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     {
       "uuid": "57b76aba-a485-464d-84ba-e9a445b25be6",
       "slug": "bob",
-      "core": false,
+      "core": true,
```
Comment (ErikSchierboom) [inline config.json]: I think I'd like to see bob moved down a bit as I think it is harder than `sum-of-multiples` and `leap`

**Comment by ricemery** [comment]
Hey @ErikSchierboom .

Thanks. I made the changes you requested.

**Comment by ErikSchierboom** [comment]
Great! The updated track should be live in a couple of minutes on https://v2.exercism.io/tracks/scala/exercises

**Comment by ricemery** [comment]
Thanks!

**Pull request merged**